to-headers/index.js

```diff
@@ -22,3 +22,5 @@ module.exports = (headers) => {
 
     return headers;
 };
+
+const supportsHeaders = () => typeof Headers === 'function';
```

Comment:
Why is it defined as a function and not a constant?
Is it likely to change between calls to this module?

Reply:
Headers is a built in constructor. Constructors are functions

Reply:
I know, I meant why is `supportsHeaders` a function that calculates this every time this module gets called?
It could be a constant defined at the top of the file:
```js
const supportsHeaders = typeof Headers === 'function';
```

Reply:
Yeah, good point. I'll do that